ffmpeg/__init__.py

```diff
@@ -74,29 +74,60 @@ class _FilterNode(_Node):
     def _get_filter(self):
         raise NotImplementedError()
 
+    def _get_params_from_dict(self, d):
+        params = ""
+        for k in self.kwargs:
+            params += k + "={}:".format(self.kwargs[k])
+        if len(params) > 0:
+            params = params[:-1]
+        return params
+
+    def _get_params_from_list(self, l):
+        return ":".join(["{}".format(i) for i in l])
+
+    def _get_filter_from_dict(self, d):
+        p = self._get_params_from_dict(d)
+        if len(p) > 0:
+            return self.NAME + "=" + p
+        return self.NAME
```

Comment:
Tends to be more obvious (IMO) that you're not mutating variables on `self`. Not a big deal - more of just a small code organizational thing